views/emails/congratulatoryMssEmail.html

```diff
@@ -0,0 +1,155 @@
+<html lang="en" style="margin:0;padding:0">
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <meta name="format-detection" content="telephone=no" />
+  <title>Subject</title>
+  <style type="text/css"> @media screen and (max-width: 480px) {
+    .mailpoet_button {width:100% !important;}
+  }
+  @media screen and (max-width: 599px) {
+    .mailpoet_header {
+      padding: 10px 20px;
+    }
+    .mailpoet_button {
+      width: 100% !important;
+      padding: 5px 0 !important;
+      box-sizing:border-box !important;
+    }
+    div, .mailpoet_cols-two, .mailpoet_cols-three {
+      max-width: 100% !important;
+    }
+  }
```

Comment:
Some of those classes are not used in the content: `.mailpoet_button`, `.mailpoet_cols-three`.  
It seems to be the default styles we use for all emails. I see the same in https://github.com/mailpoet/mailpoet/blob/master/views/emails/statsNotificationLayout.html#L8
So I guess we could somehow include those or maybe even create a common layout for all fancy HTML email. What do you think?